CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ### Added
 - Added announcement.isTombstoneableType() for testing if a given announcement type can be tombstoned
+- Added danger package for extra CI tests
```

Comment:
Ironically. I don't think this should be part of the changelog.

Reply:
Removed